tests/Controller/CkeditorAdminControllerTest.php

```diff
@@ -132,9 +134,11 @@ private function configureCRUDController()
         $this->request->get('uniqid')->shouldBeCalled();
         $this->container->get('sonata.admin.pool')->willReturn($pool->reveal());
         $this->container->get('sonata.admin.breadcrumbs_builder')->willReturn($breadcrumbsBuilder->reveal());
+        $this->container->get('admin_code.template_registry')->willReturn($templateRegistry->reveal());
```

Comment:
These tests break because we mock things that we don't own, and this fix adds more things to mock… it would be great to avoid that if we can.